packages/uploadthing/src/internal/handler.ts

```diff
@@ -62,6 +61,9 @@ const fileCountLimitHit = (
 export type RouterWithConfig<TRouter extends FileRouter> = {
   router: TRouter;
   config?: {
+    /**
+     * @deprecated this option is deprecated and will be removed in a future version
```

Comment:
since this will not work, should we put an early escape to throw if using a deprecated input, or should we allow it to work and warn in the console?

Reply:
why wont it work?

Reply:
🤦I just can't read. You're right.